fragments/arguments.sh

```diff
@@ -1,4 +1,9 @@
-# MARK: check minimal macOS requirement
+# MARK: 3 Arguments
+
+# Start trapping the script
+trap_with_arg func_trap SIGHUP SIGINT SIGQUIT SIGABRT SIGTERM SIGTSTP
```

Comment:
I think it would be easier and clearer to not use the `trap_with_arg` function here, but:

```
for sig in ( SIGHUP SIGINT SIGQUIT SIGABRT SIGTERM SIGTSTP ); do
    trap "func_trap $sig" $sig
done
```

the added abstraction through the extra function is confusing to me